index.html

```diff
@@ -1,37 +1,39 @@
 <!DOCTYPE html>
 <html lang="en">
     <head>
-        <meta charset="UTF-8" />
-        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
         <link rel="preconnect" href="https://fonts.googleapis.com">
         <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
         <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,500;1,400&display=swap" rel="stylesheet">
-        <link rel="preconnect" href="https://fonts.googleapis.com">
-        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
         <link href="https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500&family=Roboto:ital,wght@0,500;1,400&display=swap" rel="stylesheet">
         <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+        <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0" />
         <link rel="stylesheet" href="./style.css" />
+        <script src="./script.js" defer></script>
         <title>Aron Abraham Portfolio</title>
     </head>
     <body>
-
-        <!-- Header of the page with navigation bar -->
         <header class="top-navigation">
-            <a href="#" class="log">Aron Abraham</a>
+            <a href="#about" class="log">Aron Abraham</a>
             <nav class="item-navigation">
-                <ul class="frame-navigation">
-                    <li class="item-frame-nav"><a href="#portfolio-link">Portfolio</a></li>
-                    <li class="item-frame-nav"><a href="#about-link">About</a></li>
-                    <li class="item-frame-nav"><a href="#contact-link">Contact</a></li>
-                    <li class="item-frame-nav"><a href="#email-link" aria-label="email envelope"><i class="fa fa-envelope-o" ></i></a></li>
+                <div class="hamburger-menu">
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                </div>
```

Comment:
- [ ] As requested by the previous reviewer, kindly follow the Figma design.

![image](https://user-images.githubusercontent.com/81932478/174706513-1bd535f3-f775-498a-a6b2-24f6de99b900.png)
- Kindly ensure the `X` button is properly aligned with the other items in the design as well as the `menu` bar in mobile view.